docs/database/database-metrics.md

```diff
@@ -106,6 +106,8 @@ of `[ 0.001, 0.005, 0.01, 0.05, 0.1, 0.5, 1, 5, 10 ]`.
 | [`db.operation.name`](/docs/attributes-registry/db.md) | string | The name of the operation or command being executed. [5] | `findAndModify`; `HMSET`; `SELECT` | `Conditionally Required` [6] | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
 | [`error.type`](/docs/attributes-registry/error.md) | string | Describes a class of error the operation ended with. [7] | `timeout`; `java.net.UnknownHostException`; `server_certificate_invalid`; `500` | `Conditionally Required` If and only if the operation failed. | ![Stable](https://img.shields.io/badge/-stable-lightgreen) |
 | [`server.port`](/docs/attributes-registry/server.md) | int | Server port number. [8] | `80`; `8080`; `443` | `Conditionally Required` [9] | ![Stable](https://img.shields.io/badge/-stable-lightgreen) |
+| [`thread.id`](/docs/attributes-registry/thread.md) | int | Current "managed" thread ID (as opposed to OS thread ID). | `42` | `Conditionally Required` If available | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
```

Comment:
a single database operation can run on multiple threads. 

could you help me understand how it can be helpful/used on metrics?

Reply:
Hi @lmolkova , thank you very much for the insightful comment.
In such cases an operation running multithread (which I did not know about and makes total sense now that I think about it), `thread.id` would not be included in the metric. In contrast, MySQL for example provide the thread ID in its performance log and the ID is often used for troubleshooting of slow/blocked queries I believe. I found a good example of using thread IDs to handle a lock wait timeout: https://severalnines.com/blog/how-fix-lock-wait-timeout-exceeded-error-mysql/ Also for some simpler cases, it could be useful if you can aggregate the stats on the thread ID to identify any meaningful performance differences between the threads.

Even with all that in mind, it might be too much to put it in "Conditionally Required" requirement level considering the use cases being relatively specific. If it becomes a question about "how often adopted does something need to be included in the spec?", we are very comfortable with leaving the decision up to the community whether this attribute qualifies and dropping it if that makes more sense.

Reply:
@lmolkova any further thoughts on this?